src/angular/projects/spark-angular/src/lib/directives/inputs/sprk-input/sprk-input.directive.ts

```diff
@@ -16,9 +16,9 @@ export class SprkInputDirective implements OnInit {
   OnChange($event) {
```

Comment:
need to remove ($event) from OnChange or change the @HostListener to supply it, like @HostListener('change', ['$event']). Is the arguments are mismatched the aot compiler will error out